src/Simulation/Simulators.Tests/TestProjects/IntrinsicTests/Bitwise/Tests.qs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Tests {
+    open Microsoft.Quantum.Bitwise;
+    open Microsoft.Quantum.Diagnostics;
+
+    @Test("QuantumSimulator")
+    function XorIsCorrect() : Unit {
+        EqualityFactI(199, Xor(248, 63), "Xor was incorrect.");
```

Comment:
Nit: because the test results are often reviewed by humans for both passing and failing tests on my previous projects we used to word the assert messages in terms of what is expected, then passing test ("green") would read as the expectation that was met and failing test ("red") would read as the expectation that wasn't met. It removed the cognitive dissonance when reviewing test logs with "Xor was incorrect." messages while still passing :) Instead, the message would be more like "Bitwise Xor of 11111000 and 00111111 should be 11000111"

Reply:
Thanks for raising that! In this case, `EqualityFactI` uses `FormattedFailure` to print the expected and actual values for each failing test, such that I think I the best way to incorporate that feedback is to add the arguments to the message:

```suggestion
        EqualityFactI(199, Xor(248, 63), "Xor(248, 63) was incorrect.");
```

Reply:
My concern was mostly with having _negative_ message printed when the test is actually passing. Your call.

Reply:
Ah, got it; sorry for misunderstanding. Here, the string argument is the message to be printed in the case of failure, while the name of each test is taken from the name of the callable implementing the test. Thus here, `"Xor(248, 63) was incorrect."` only gets printed in the negative case, while `☑ XorIsCorrect` should be printed in the positive case.